src/devtools.ts

```diff
@@ -50,9 +50,10 @@ export function preactDevtoolsPlugin({
 				code = `import "${source}";\n${code}`;
 
 				log(`[inject] ${kl.cyan(source)} -> ${kl.dim(id)}`);
+				return code;
 			}
 
-			return code;
+			return undefined;
```

Comment:
Let's just drop the whole return statement here as `undefined` is the default value returned from functions.